chapter-06/inprogress/bookworm.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+	"sort"
+)
+
+// A Bookworm contains the list of books of a specific person.
+type Bookworm struct {
```

Comment:
I would probably use `Reader` as the name or something. `Bookworm` is cute, but it can be rather confusing, if you start reading a random func.

Reply:
In our first version, it was named `Reader`, we changed it to `Bookworm` to avoid confusion with the Interface `Reader` from Go... 
@ablqk @floppyzedolfin wdyt? Should we keep going with `Bookworm`?

Reply:
Hmm. Yeah, in that case, let's leave it as `Bookworm`.